docs/prepare-a-shared-rp-development-environment.md

```diff
@@ -228,6 +228,24 @@ locations.
 
    TODO: more steps are needed to configure aro-v4-portal-shared.
 
+1. Create an AAD application which will fake up the dbtoken client.
+
+   1. Create the application and set `requestedAccessTokenVersion`
+
+   ```bash
+   AZURE_DBTOKEN_CLIENT_ID="$(az ad app create --display-name dbtoken \
+      --oauth2-allow-implicit-flow false \
+      --query appId \
+      -o tsv)"
+
+   OBJ_ID="$(az ad app show --id $AZURE_DBTOKEN_CLIENT_ID --query objectId)"
+
+   # TODO - currently the az-cli cannot update the requestedAccessTokenVersion
```

Comment:
I would remove this TODO: a) not clear what one needs to do here and b) `az rest` is ok.

P.S.: `az rest` is part of az cli :)